qemu/tests/block_hotplug.py

```diff
@@ -69,8 +58,32 @@ def unplug_device(vm, get_disk_cmd, device):
                                             (vm, get_disk_cmd), disks_before_unplug)) != 0, pause)
         return unplug_status
 
+    def con_devs_dict(vm, drive_format, device, con_devs, dev_list):
```

Comment:
If I understand this properly it's here to search for `controller` in case of `scsi-hd` is used. Well I think there are simpler ways of doing so (like we don't need to distinguish between devices and controllers and simply keep removing added devices in the reverse order). Anyway if you want to keep doing this way please improve the docstring so people don't have to follow the code and usage to get the meaning.